visualization/dashboard/src/MLIDashboard/JointDataset.ts

```diff
@@ -176,6 +176,8 @@ export class JointDataset {
                 sortedCategoricalValues: args.metadata.modelType !== ModelTypes.regression ? args.metadata.classNames : undefined,
                 category: ColumnCategories.outcome
             };
+            const predictions = args.predictedY
```

Comment:
what is the plan for these pieces? There are ways of getting the predictedYs off of a cohort (if they exist). but predictedY can be undefined, so the line below this will break then. And in the case of a classifier, averaging isn't really defined. We can discuss why this is needed, and where the calculation should live.

Reply:
I made an attempt to add average and probably forgot to clean it up later. As discussed below - removing this since we are not adding average now.